src/cdp/cdp.zig

```diff
@@ -303,6 +303,9 @@ pub fn BrowserContext(comptime CDP_T: type) type {
         node_registry: Node.Registry,
         node_search_list: Node.Search.List,
 
+        // Holds the data to be able recreate the fake isolated world, see: @isolated_world
+        fake_isolatedworld: ?@import("domains/runtime.zig").ExecutionContextCreated,
```

Comment:
Not against having a `src/cdp/execution_context.zig`  with a `Created` struct inside. Also can keep as-is.